scripts/apps/authoring/authoring/services/AuthoringService.js

```diff
@@ -230,6 +230,11 @@ export function AuthoringService($q, $location, api, lock, autosave, confirm, pr
     this.publish = function publish(orig, diff, action = 'publish') {
         let extDiff = helpers.extendItem({}, diff);
 
+        // if current document is image and it has been changed on 'media edit' we have to update the etag
+        if (orig.type === 'picture') {
```

Comment:
Why is this conditional? Shouldn't etag work the same for every type?

Reply:
because differentiation of etags can be only if there is some change on image with media editor. You can't change brightness and contrast on video (for now) and audio files

Reply:
If I understand correctly it would work fine even without the conditional. It's only that other types of files can't take advantage of it yet. Could you remove the conditional and add a comment describing the case when it's necessary?

Reply:
here you are

Reply:
I needed to add check if etag exists on diff, because there are some cases that etag is not send through diff.